README.md

```diff
@@ -105,6 +109,35 @@ see the [cheatsheet](https://github.com/turion/rhine/blob/master/CHEATSHEET.md).
 * https://github.com/turion/rhine-tutorial: Presentation and tutorial app
 * https://github.com/turion/sonnendemo: Demo application
 
+
+## FAQ
+
+* Why does my blocking code, eg `arrMCl readLn`, behave [erratically](https://github.com/turion/rhine/issues/153)?
+
+Clocks must be the only thing that blocks the thread, not `ClSF`s. So for example, you can fix:
+
+```haskell
+arrMCl readLn
+```
+
+by using:
+
+```haskell
+tagS >>> arr read :: ClSF IO StdinClock () Int
+```
+
+`tagS` contains the string that the `StdinClock` grabbed from `stdin`, and only the clock has been allowed to block the thread!
```

Comment:
```suggestion
[`tagS`](https://hackage.haskell.org/package/rhine/docs/FRP-Rhine.html#v:tagS) contains the string that the [`StdinClock`](https://hackage.haskell.org/package/rhine/docs/FRP-Rhine.html#t:StdinClock) grabbed from `stdin`, and only the clock has been allowed to block the thread!
```

I like to have links to reference docs, because few people will know all the Rhine components by heart.

Reply:
These links could of course be applied everywhere in the documentation.